keylime/cloud_verifier_tornado.py

```diff
@@ -965,7 +997,7 @@ async def invoke_get_quote(
                 rmc.record_create(agent, json_response, ima_policy)
 
             failure = cloud_verifier_common.process_quote_response(
-                agent, ima_policy.ima_policy, json_response["results"], agentAttestState
```

Comment:
Why was this `.ima_policy` removed? Should it also have a similar `verifier_read_policy_from_cache` call, like in `process_agent`?

Reply:
I realized, while writing the "Durable Attestation" (DR) code that the only thing needed from `ima_policy` DB object was the (potentially long) string from the `ima_policy` column on the `allowlists` table. Since `cloud_verifier_common.process_quote_response` is reused by DR, just passing a string made both codepaths significantly simpler.